Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
+  url "https://github.com/boot-clj/boot/releases/download/2.0.0-rc4/boot.sh",
+      :using => :nounzip
+  sha1 "68b9b7409579ae16f482b4fd6bde31b74a7d2d76"
+
+  def install
+    mv "boot.sh", "boot"
+    bin.install Dir['*']
+  end
+
+  test do
+    system "#{bin}/boot -V"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!